THIRD-PARTY-NOTICES.md

```diff
@@ -0,0 +1,7 @@
+The Uno Platform uses third-party libraries and resources that may be distributed under licenses different from the Uno Platform software.
+In the event of accidental failure to list a required notice, please bring it to our attention. Post an issue at :
+
+	https://github.com/unoplatform/uno/issues
+
+The attached notices are provided for information only.
```

Comment:
Please add the notice :)

```suggestion
The attached notices are provided for information only.

UnoContoso sample
-----------------------

The UnoContoso sample is based on Microsoft's "Customers Orders Database sample" UWP sample app: https://github.com/microsoft/Windows-appsample-customers-orders-database

The MIT License (MIT)

Copyright (c) 2016 Microsoft

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.


```

Reply:
Thank you ;)